moveit_ros/visualization/rviz_plugin_render_tools/include/ogre_helpers/mesh_shape.hpp

```diff
@@ -0,0 +1,140 @@
+/*
```

Comment:
Where does this come from?

Reply:
It comes from Rviz project, but it's not ported to ROS2, I'll open a porting PR today

Reply:
Please add a NOTE/TODO mentioning that here